OrbitQt/orbitmainwindow.cpp

```diff
@@ -545,6 +548,7 @@ void OrbitMainWindow::on_actionOpen_Preset_triggered() {
     }
     break;
   }
+  ui->actionSave_Preset_As->setDisabled(false);
```

Comment:
I am a bit confused, could you explain why this is the correct place to enable preset saving?
Same for the Capture below?
It feels a bit unintuitive.

Reply:
When you open a preset or a capture, you can then save it again. The other place where we enable preset saving is the capture_finished_callback above, which is called on stopping the capture. 

Regarding presets the situation is also that you can open a preset without actually having capture data.

I'll add comments to clarify if you agree this all makes sense.

Reply:
I am unsure about all possible situations that can happen. For example. 

* I start Orbit, load a module, hook 5 functions. Now I want to save this as preset (before taking a capture) Is this possible?
* I start Orbit, load a capture from file which has hooked functions and a frame track. I want to save this as a preset. Is this possible?

Reply:
I actually thought about the same. Shouldn't the disable/enable capture be more related to the "HasCaptureData" Property and the disable/enable preset more to the "Are Functions Hooked"?

Reply:
The callbacks for capture data are there (OnCaptureCleared(), etc.), but for presets not. I think it would actually be good to separate out the preset changes to the menu from this PR. For saving presets, CaptureData is not relevant and it work whether that exists or not. WDYT?

Reply:
Sounds good, can you create a bug for the preset change?

Reply:
b/173679796